### DISCUSSION TITLE
/.well-known/openid-configuration issuer contains current sessions port number

### DISCUSSION THREAD

**Discussion by 8321876218** [goauthentik/authentik#7885] [Q&A] [answered]
A little about my setup:
-authentik is running in Docker on a Windows Server host
-I am using plain nginx as my reverse proxy

The issue I am running into currently is when I go to `https://sub.domain.tld/application/o/<app>/.well-known/openid-configuration`, "issuer" shows as `https://sub.domain.tld:<session_port_number>/application/o/<app>/` which I believe is causing the "Issuer name does not match authority" errors I'm getting on my application.

Does anyone have any direction to point me to getting this resolved? As far as I am aware the port number being listed is not proper, but I have no idea what to do to remove it.

**Comment by BeryJu** [comment]
Compare your nginx config to this one: https://goauthentik.io/docs/installation/reverse-proxy; especially make sure that the `Host` header does not include the port

**Reply by 8321876218** [reply, marked as answer]
Awesome, thanks for confirming the issue was with nginx. I had `proxy_set_header X-Forwarded-Host $host:$remote_port;`, I changed it to `proxy_set_header X-Forwarded-Host $host;` and now everything looks good! Thanks for the help and the quick response
